package.json

```diff
@@ -69,6 +69,8 @@
   "devDependencies": {
     "@babel/core": "^7.12.0",
     "@babel/preset-env": "^7.1.6",
+    "@babel/preset-react": "^7.22.5",
+    "@react-native-community/cli": "^11.3.2",
```

Comment:
Are these still needed?

Reply:
I removed them in a1cbaebb15b6f3fd69017fcc2b5c983afa7ef161 and also bumped `@babel/core` and `@babel/preset-env` since I was getting
> Duplicate __self prop found. You are most likely using the deprecated transform-react-jsx-self Babel plugin.

without them.